lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -67,7 +67,6 @@
                     // name is being routed to.
                     router = this.router = new Backbone.Router();
                     _.each([
-                        [':default', _.bind(this.routeNotFound, this)],
```

Comment:
@clrux This is the default router so it is absolutely needed. All routers must be able to handle arbitrary routes. I think the problem is that it is treating ``#main`` as a route, not as a link to an anchor tag. There seem to be a variety of solutions for this:

http://stackoverflow.com/questions/12081894/backbone-router-navigate-and-anchor-href
https://gist.github.com/philfreo/9240297

Unfortunately Dan and Peter wrote this code, neither of whom is with us any more. Maybe @bjacobel has some experience with how to handle this.